plugins/topology/README.md

```diff
@@ -207,6 +207,27 @@ The Topology plugin enables you to visualize the workloads such as Deployment, J
     backstage.io/kubernetes-id: <BACKSTAGE_ENTITY_NAME>
   ```
 
+  If you have multiple entities and Red Hat Dev Spaces is configured, you can add the `backstage.io/kubernetes-label-selector` annotation to the `catalog-info.yaml` file.
+
+  ```yaml title="catalog-info.yaml"
+  annotations:
+    backstage.io/kubernetes-label-selector: 'component in (<BACKSTAGE_ENTITY_NAME>,che)'
+  ```
+
+  If you are using the previous label selector ensure that you add the following labels to your resources so that the Kubernetes plugin gets the Kubernetes resources from the requested entity:
```

Comment:
```suggestion
  If you are using the previous label selector, ensure that you add the following labels to your resources so that the Kubernetes plugin gets the Kubernetes resources from the requested entity:
```